docs/resources/User.md

```diff
@@ -14,6 +14,22 @@ data:image/jpeg;base64,BASE64_ENCODED_JPEG_IMAGE_DATA
 
 Ensure you use the proper header type (`image/jpeg`, `image/png`, `image/gif`) that matches the image data being provided.
 
+## Usernames and Nicknames
+
+Discord enforces the following restrictions for usernames and nicknames:
```

Comment:
this is intentional. guild nicknames do not currently, but channel nicknames do.. it's just something we overlooked but isn't a critical bug.